docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic.md

```diff
@@ -5,224 +5,12 @@ sidebar_label: "YData-Synthetic"
 sidebar_custom_props: { icon: 'img/integrations/ydata_synthetic_icon.png' }
 ---
 
-import Prerequisites from './components/deployment_pattern_prerequisites.jsx'
-import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+YData-Synthetic is an open-source synthetic data engine. Using different kinds of Generative Adversarial Networks (GANS), the engine learns patterns and statistical properties of original data. It can create endless samples of synthetic data that resemble the original data.
 
-import InProgress from '/docs/components/warnings/_in_progress.md'
+:::info 
 
-<InProgress />
+To use Great Expectations (GX) with YData-Synthetic, see [How to use GX with YData-Synthetic](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic) in version 0.15.50 of the documentation.
 
-_This piece of documentation was authored by [Arunn Thevapalan](https://www.linkedin.com/in/arunn-thevapalan/)._
+For more information about implementing and using Ydata-Synthetic, see the [YData-Synthetic site](https://github.com/ydataai/ydata-synthetic).
```

Comment:
```suggestion
For more information about implementing and using Ydata-Synthetic, see the [YData-Synthetic site](https://github.com/ydataai/ydata-synthetic).
```
The previous instance was YData. Confirm the correct format and use it consistently.